edx_rest_api_client/client.py

```diff
@@ -71,7 +73,7 @@ def get_oauth_access_token(cls, url, client_id, client_secret, token_type='beare
 
     def __init__(self, url, signing_key=None, username=None, full_name=None, email=None,
                  timeout=5, issuer=None, expires_in=30, tracking_context=None, oauth_access_token=None,
-                 session=None, jwt=None, **kwargs):
+                 session=None, jwt=None, retry_attempts=0, **kwargs):
```

Comment:
Could the default be None?  Could it then use some type of configuration to determine the default if it is not passed?

Reply:
I set the default to none. I wasn't sure what you mean by configuration for the default. Wouldn't the default just be not to retry?